tests/flow/test_pagerank.py

```diff
@@ -17,30 +17,30 @@ def __init__(self):
         redis_graph = Graph(GRAPH_ID, redis_con)
 
     def test_pagerank(self):
-        # Pagerank considers only nodes of given label and 
+        # Pagerank considers only nodes of given label and
         # relation of given relationship type
         # multiple edges between two nodes are considered as a single connection.
         queries = [
             # Single Label, single connection.
-            "CREATE (a:L {v:1})-[:R]->(b:L {v:2})"
+            "CREATE (a:L {v:1})-[:R]->(b:L {v:2})",
             # Single Label, multi connection.
-            "CREATE (a:L {v:1})-[:R]->(b:L {v:2}), (a)-[:R]->(b)"
+            "CREATE (a:L {v:1})-[:R]->(b:L {v:2}), (a)-[:R]->(b)",
             # Multi Label, single connection.
-            "CREATE (a:L {v:1})-[:R]->(b:L {v:2}), (:X)-[:R]->(:X)"
+            "CREATE (a:L {v:1})-[:R]->(b:L {v:2}), (:X)-[:R]->(:X)",
```

Comment:
The changes in this file are not required by this fix, but previously all of these strings were implicitly concatenated into a single query.

Reply:
good catch